File: Server/src/main/content/global/skill/production/cooking/recipe/pie/impl/RedberryPie.java
package content.global.skill.production.cooking.recipe.pie.impl;

import content.global.skill.production.cooking.recipe.pie.PieRecipe;
import core.game.node.item.Item;

/**
 * The Redberry pie.
 */
public class RedberryPie extends PieRecipe {

    private static final Item UNCOOKED_PIE = new Item(2321);

    private static final Item REDBERRIES = new Item(1951);

    @Override
    public Item getProduct() {
        return UNCOOKED_PIE;
    }

    @Override
    public Item[] getIngredients() {
        return new Item[]{REDBERRIES};
    }

}

File: Emulator/src/main/content/global/skill/production/cooking/recipe/pie/impl/RedberryPie.kt
package content.global.skill.production.cooking.recipe.pie.impl

import content.global.skill.production.cooking.recipe.pie.PieRecipe
import org.rs.consts.Items
import core.game.node.item.Item

/**
 * Redberry pie.
 */
class RedberryPie : PieRecipe() {

    override fun getProduct(): Item {
        return UNCOOKED_PIE
    }

    override fun getIngredients(): Array<Item> {
        return arrayOf(REDBERRIES)
    }

    companion object {
        private val UNCOOKED_PIE = Item(Items.UNCOOKED_BERRY_PIE_2321)
        private val REDBERRIES = Item(Items.REDBERRIES_1951)
    }
}
